schemas/iso19139.che/src/main/plugin/iso19139.che/index-fields/language-default.xsl

```diff
@@ -433,10 +437,12 @@
         <xsl:variable name="address" select="string-join(gmd:contactInfo/*/gmd:address/*/(
                                       gmd:deliveryPoint|gmd:postalCode|gmd:city|
                                       gmd:administrativeArea|gmd:country)/gco:CharacterString/text(), ', ')"/>
-
+        <xsl:variable name="uuid" select="@uuid"/>
+        <xsl:variable name="position" select="0"/>
```

Comment:
no need for the variable ? should we use it ?

Reply:
True. We are not using position in geocat.